src/tpm2_wrap.c

```diff
@@ -172,6 +172,249 @@ int wolfTPM2_Init(WOLFTPM2_DEV* dev, TPM2HalIoCb ioCb, void* userCtx)
     return rc;
 }
 
+#ifdef WOLFTPM2_ENABLE_CSHARP
```

Comment:
This project has mostly avoiding using heap, but I understand the need. These new API's can be enabled always if HEAP use is allowed. I would like to add a new API's `WOLFTPM2_NO_HEAP` and just wrap if `#ifndef WOLFTPM2_NO_HEAP`.

Reply:
I think I can do that. should be simple. So, anytime I use XMALLOC and XFREE I would wrap it with  !WOLFTPM2_NO_HEAP